lib/rspec/abq/formatter.rb

```diff
@@ -82,6 +87,27 @@ def self.abq_result(example)
           end
         end
       end
+
+      private_class_method def self.rspec_retry_attempts(example)
+        return unless defined?(RSpec::Retry)
+        return unless example.metadata.key?(:retry_attempts)
+        return unless example.metadata.key?(:retry_exceptions)
+
+        retry_attempts = example.metadata[:retry_attempts]
+        retry_exceptions = example.metadata[:retry_exceptions]
+        return unless retry_attempts > 0
+
+        retry_attempts.times.map do |attempt_index|
+          {
+            status: {type: :failure},
+            id: example.id,
+            display_name: example.metadata[:full_description],
+            output: retry_exceptions[attempt_index],
+            runtime: 0, # unknown
+            meta: {}
```

Comment:
Is there any metadata on the rspec retry attempts? If not a comment be added that this is intentionally empty

Reply:
Nope. I'll drop a comment in here 👍